index.js

```diff
@@ -1,18 +1,233 @@
+        if (count === 1) {
+          console.log("Processing!");
+        } else if (count === 13) {
+          console.log("I think your computer's a bit slow...");
+        } else if (count === 15) console.log("DONE");
+        else {
+          console.log("...");
+        }
+      }
+      advancedCpuThingz(arr, secretNumber, 0, arr.length - 1);
+      console.log(
+        "I think my brain just EXPANDED right now dude! I know the answer is in this list! ;)"
+      );
+      stepThree()
+    }
+  }
+  //Confirmation on whether or not secret number was found from guessing function in amount of tries specified.
+  
+  async function stepThree() {
+    let newComment = await ask(`Your number's in this list! Am I right? `);
+    if (
+      (newComment === "yes") ||
+      (newComment === "Yes") ||
+      (newComment === "Yeah") ||
+      (newComment === "yeah")
+    ) {
+      console.log("Amazing!");
+      stepfour()
+    }
+    else if (console.log("Hey now don't be salty. You gave it your best shot. Don't sour it now by trying to cheat.")); {
+  stepfive()}
+  
+//Final resolution of the game. Asks user if CPU guessed number in amount of tries it claims it could.
+  stepfour ()
+    async function stepfour() {
+    let newComment4 = await ask(
+      "Did I figure it out in seven or less tries...just like I said I would ;)? "
+    );
+    if (
+      (newComment4 === "yes")
+    ) {
+      console.log(
+        `I think I may be the greatest of all time. I guessed it right! Thanks for playing friend`
+      );
+      stepfive();
+    } else
+      (console.log(
+        "I think someone MIGHT..be trying to cheat here and might ALSO...be a sore loser. Try again next time o.O "));
+    {
+      process.exit();
+    }
+      }
+    
+  }
+  //End Game Instructions. Asking user if they'd like to try again or not with options to either return to the "main menu" or try another round of this game
+
+  async function stepfive() {
+    let newComment2 = await ask("Would you like to try again? ");
+    if (
+      (newComment2 === "yes")
+    ){
+      let newComment3 = await ask(
+        "Would you like to play another round here or go back to the main menu? Enter `here` or main menu` "
+      );
+      if (newComment3 === "here") {
+        start();
+      } else if (newComment3 === "main menu") {
+        init();
+      }
+    } else {
+      console.log("Bye");;
+    }
+  }
+}
+
+//Officially starting the Reverse Number Guessing Game
+async function start2() {
+  console.log(
+    "Welcome to the Second Number Guessing Game! In THIS version, you (the human) can try to guess a secret number that I (the CPU) will think of."
+  );
+  console.log(
+    "First. Let us set some parameters before the game begins.\nI don't wanna make this too easy or too hard on you so I'll let you set the range of numbers that I have to work with."
+  );
+  let minimum = await ask("What's the lowest number you can think of? ");
+  if (minimum < 0) {

```

Comment:
this story should be in the first game where the computer is guessing the users number